pkg/apps/strategy/support/acceptor.go

```diff
@@ -72,7 +72,8 @@ func (c *acceptAvailablePods) Accept(rc *corev1.ReplicationController) error {
 		case nil:
 			return nil
 		case watch.ErrWatchClosed:
-			fmt.Fprint(c.out, "Warning: acceptAvailablePods encountered %T, retrying", watch.ErrWatchClosed)
+			//fmt.Fprint(c.out, "Warning: acceptAvailablePods encountered %T, retrying", watch.ErrWatchClosed)
```

Comment:
this looks spurious.

Reply:
I just changed that to get it to build in my env and forgot to finalize. Why it fails to build in my env I'm not sure. I'm building with go 1.11 vs. 1.7 but wouldn't think it would cause this issue.

Reply:
@deads2k I backed this change out. Not sure what caused the initial issue but I was using different version of go.